spec/ParseGeoPoint.spec.js

```diff
@@ -24,7 +24,7 @@ describe('Parse.GeoPoint testing', () => {
         });
       }
     });
-  });
+  }, 60000);
```

Comment:
What's the 60000s for?  Tried looking up jasmine docs, they're useful for examples but not much of an api reference..